controls/dataform/validation.md

```diff
@@ -89,6 +89,70 @@ You can use the following properties for error message styling and customization
 
 ![.NET MAUI DataForm Error styling](images/dataform-error-message-styling-desktop.png)
 
+## Runtime Validation
+
+If you want to add the validation runtime, when editors are generated, then you have to use the `EditorsGenerated` event. Inside the event you can define the validation. 
+
+The available validations DataForm provides are listed in the table:
+
+| DataForm Validation Rule | Definition |
+| ----- | ------ |
+| (DataForm level) `DataFormObjectValidationRule` | Represents a custom validation rule of the DataForm control. |
+| `DataFormEditorCustomValidationRule` | Represents a custom validation rule for the editor. The validation rule can execute a custom validation logic on the underlying business object.|
+| `DataFormEditorLengthValidationRule` | Validates that the string length of the current value is between the specified minimum and maximum length. |
+| `DataFormEditorRangeValidationRule` | Validates that the current value is between the specified minimum and maximum value. |
+| `DataFormEditorRegexValidationRule` | Executes a custom regular expression to validate the current string value. |
+| `DataFormEditorRequiredValidationRule` | Validates that the current value is not null or an empty string. |
+
+> When using this approach and in addition, the validation is added to the properties defined in the `ViewModel` using Data Annotation, the validation set in the `ViewModel` is with high priority (you won't be able to remove, modify the validation in the `EditorGenerated` event).
```

Comment:
I keep reading this part and I find it very hard to understand. I highly recommend rewriting it for clarity. You might need to split it into several sentences or whatever works best.